plugins/osdn/ovs/proxy.go

```diff
@@ -0,0 +1,151 @@
+package ovs
+
+import (
+	"net"
+	"sync"
+
+	"github.com/golang/glog"
+
+	"github.com/openshift/openshift-sdn/plugins/osdn"
+	"github.com/openshift/openshift-sdn/plugins/osdn/api"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	pconfig "k8s.io/kubernetes/pkg/proxy/config"
+)
+
+type ovsProxyPlugin struct {
+	registry *osdn.Registry
+	podsByIP map[string]*kapi.Pod
+	mutex    sync.Mutex
```

Comment:
Generic name `mutex` may give the impression that it protects the whole struct. Can we rename mutex so that we know it's only needed for podsByIP map.